gdx/src/com/badlogic/gdx/audio/Music.java

```diff
@@ -84,6 +84,13 @@
 	/** Returns the playback position in seconds. */
 	public float getPosition ();
 
+	/** Sets the playback rate
+	 * @param playbackRate rate: 1 => normal */
```

Comment:
I guess you want to express that `playbackRate` should be greater than equal 1?
The wording could be better in my opinion.

Additionally you should document, that some backends only support `playbackRate`s that are smaller than 4 (e.g GwtMusic).

Reply:
I wanted to say that a playback rate of 1 is the normal speed
```suggestion
	 * range: Android + Desktop 0.5 to 2, GWT 0.5 to 4
	 * @param playbackRate < 1 slower, = 1 normal, > 1 faster */
```

Reply:
```suggestion
	 * range: Android and Desktop 0.5 to 2, GWT 0.5 to 4
	 * a rate of 1 play at normal speed, 2 at twice speed and 0.5 at half speed
	 * @param playbackRate new playback rate */
```